### ISSUE TITLE
2023100301 - Tributary to Read Creek

### ISSUE THREAD

**Issue by lucy-schick** [NewGraphEnvironment/fish_passage_fraser_2023_reporting#88]
- [x] No stream present in FWA at this location OR no modelled crossing in bcfishpass. Confirm culvert doesn't link with other crossing in bcfishpass, if so unlink with  https://github.com/NewGraphEnvironment/bcfishpass/blob/main/data/pscis_modelledcrossings_streams_xref.csv

**Comment by NewGraphEnvironment** [comment]
yup - Looks like both  (no stream and therefore no crossing).  No action required I don't think except to maybe confirm after that it bcfishpass doesn't tie the crossing to read creek where there is a bridge

![image](https://github.com/NewGraphEnvironment/fish_passage_fraser_2023_reporting/assets/46538103/2eb299c7-7729-448b-9206-f3e8364491c5)

**Comment by lucy-schick** [comment]
crossing is not tied to another bcfishpass crossing. Success
